Explorer/Assets/DCL/SDKComponents/SceneUI/Tests/UITransformInstantiationSystemShould.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Arch.Core;
+using Cysharp.Threading.Tasks;
+using DCL.SDKComponents.SceneUI.Components;
+using DCL.SDKComponents.SceneUI.Systems.UITransform;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+namespace DCL.SDKComponents.SceneUI.Tests
+{
+    public class UITransformInstantiationSystemShould : UITransformSystemTestBase<UITransformInstantiationSystem>
+    {
+        [SetUp]
+        public async void SetUp()
+        {
+            await base.Initialize();
```

Comment:
have you tried to return `Task` instead of `UniTask` from that method? Maybe it will fix the bug mentioned below.
Also everything that is connected to Addressables is better to move to `PlayMode`, just use the common assembly "DCL.PlayMode.Tests" and connect to it via `asmref`. Don't create too many assemblies as Unity can blow up (it happened in `unity-renderer`)

Reply:
I've tried to return `Task` instead of `UniTask` and the issue still happens.

On another hand, as you suggested, I've made the tests belong to `PlayMode` 👍 

Done here: https://github.com/decentraland/unity-explorer/pull/204/commits/dfa907088b280335f2541eb242807ecf66140ca5